packages/familie-form-elements/package.json

```diff
@@ -30,5 +30,6 @@
   },
   "peerDependencies": {
     "react": "^16.12.0"
-  }
+  },
+  "gitHead": "ad82cdfee8d411a14bb5748eabe719baa1f2d881"
```

Comment:
Av nysgjerrighet, hva er denne linja for noe? aldri sett før 🙈

Reply:
Kan fjernes, kom fra canary-bygget :)